cmd/fortifyExecuteScan.go

```diff
@@ -965,12 +967,14 @@ func determinePullRequestMergeGithub(ctx context.Context, config fortifyExecuteS
 	email := ""
 	options := github.PullRequestListOptions{State: "closed", Sort: "updated", Direction: "desc"}
 	prList, _, err := pullRequestServiceInstance.ListPullRequestsWithCommit(ctx, config.Owner, config.Repository, config.CommitID, &options)
```

Comment:
prList, _, err := 
the _ would refer to the response, would it make sense that the response is not nil? I had the same once with the Splunk hook...

Reply:
The last commits at least address the user lookup issue as my test revealed.